subscription-service/src/main/kotlin/com/egm/stellio/subscription/service/SubscriptionService.kt

```diff
@@ -123,12 +122,15 @@ class SubscriptionService(
 
             databaseClient.sql(
                 """
-                INSERT INTO geometry_query (georel, geometry, coordinates, pgis_geometry, subscription_id) 
-                VALUES (:georel, :geometry, :coordinates, ST_GeomFromText(:wkt_coordinates), :subscription_id)
+                INSERT INTO geometry_query (georel, geometry, coordinates, pgis_geometry, geoproperty, subscription_id) 
+                VALUES (
+                    :georel, :geometry, :coordinates, ST_GeomFromText(:wkt_coordinates), :geoproperty, :subscription_id
+                )
```

Comment:
what about existing data? do we need a migration script to populate the `geoproperty` column?